src/js/content.js

```diff
@@ -2,14 +2,18 @@
 // is running]
 
 // Listen for messages from the page to do smart contract invocations.
-// TODO: this should first do a test to determine gas cost and THEN do send
 window.addEventListener('message', (event) => {
   // We only accept messages from ourselves
   if (event.source !== window) {
     return
   }
 
-  if (event.data.type && (event.data.type === 'FROM_PAGE')) {
+  if (event.data.type && (event.data.type === 'NEOLINK_GET_EXTENSION_STATUS')) {
```

Comment:
It might be worth destructuring the event object at the top of the listener  to clean up all of the instances of `event`. In addition, you would be able to remove all of these extra `event.data.type && `s (I think)